api/Element.json

```diff
@@ -6098,6 +6098,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options_unadjustedMovement_parameter": {
+          "__compat": {
+            "description": "<code>options.unadjustedMovement</code> parameter",
+            "support": {
+              "chrome": {
+                "version_added": "81"
```

Comment:
Per https://bugs.chromium.org/p/chromium/issues/detail?id=1215235 this was accidentally not shipped, and per https://storage.googleapis.com/chromium-find-releases-static/a21.html#a21641ea63a9abcdc06670adf58ccb6a0fb2905a it seems like it finally happened in 92.